common-app-lib/src/cmr/common_app/data/search/collection_for_gran_acls_caches.clj

```diff
@@ -81,8 +81,25 @@
                                                   :collection
                                                   elastic-item))
                           :concept-id (:_id elastic-item)))
-       query (q-mod/query {:concept-type :collection
-                           :condition q-mod/match-all
+       query (qm/query {:concept-type :collection
+                           :condition qm/match-all
+                           :skip-acls? true
+                           :page-size :unlimited
+                           :result-format :query-specified
+                           :result-fields (cons :concept-id acl-rhh/collection-elastic-fields)
+                           :result-features {:query-specified {:result-processor result-processor}}})]
+  (:items (qe/execute-query context query))))
+
+(defn- fetch-collection
+ "fetch one collection from elastic search"
+ [context collection-concept-id]
+ (let [result-processor (fn [_ _ elastic-item]
+                         (assoc (util/delazy-all (acl-rhh/parse-elastic-item
+                                                  :collection
+                                                  elastic-item))
+                          :concept-id (:_id elastic-item)))
+       query (qm/query {:concept-type :collection
+                           :condition (qm/string-condition :concept-id collection-concept-id)
```

Comment:
These need to line up to the left by 4 spaces. 
:concept-type
:condition 
:skip ... all should line up.